images/monitoring/Dockerfile

```diff
@@ -21,10 +21,27 @@ ENV TOOL_NAME monitoring
 # Temporarily add test-infra to the image to build custom tools
 ADD ./tools $TEMP_TOOLS_REPO_DIR
 
+# These go dependencies need to be included before cloud mail files to avoid stale checkout due to
+# no .git file in cloud mail
+RUN go get -u cloud.google.com/go
+RUN go get -u google.golang.org/genproto/googleapis/api/annotations
+RUN go get -u google.golang.org/genproto/googleapis/iam/v1
+RUN go get -u google.golang.org/genproto/protobuf/field_mask
+
+#TODO(yt3liu): Use the alpha released after the go library builds correctly with the latest release
```

Comment:
do we have any update on this?

Reply:
Let's use the one in knative-monitoring for now until a more stable release is out.